adm_program/system/function.php

```diff
@@ -697,6 +697,48 @@ function admFuncGetDirectoryEntries($directory, $searchType = 'file')
 }
 
 /**
+ * Gets the total, free and used disk space.
+ * @param string $path Path of the filesystem
+ * @return array<string,int>
+ */
+function admGetDiskSpace($path = '/')
+{
+    $total = disk_total_space($path);
+    $free = disk_free_space($path);
+    $used = $total - $free;
+
+    return array('total' => $total, 'free' => $free, 'used' => $used);
+}
+
+/**
+ * Returns a human readable size with unit
+ * @param int  $size The size in bytes
+ * @param bool $si   Use SI or binary unit. Set true for SI units
+ * @return string
```

Comment:
Please add a description here how the array will be filled with each value. Maybe additional an example.

Reply:
which array? I think you mean the format of the return string

Reply:
ok, copy paste thing. I mean the return string.